mayastor/src/core/reactor.rs

```diff
@@ -54,11 +48,11 @@ use spdk_sys::{
 
 use crate::core::{Cores, Mthread};
 
-pub(crate) const INIT: usize = 1 << 1;
-pub(crate) const RUNNING: usize = 1 << 2;
-pub(crate) const SHUTDOWN: usize = 1 << 3;
-pub(crate) const SUSPEND: usize = 1 << 4;
-pub(crate) const DEVELOPER_DELAY: usize = 1 << 5;
+pub(crate) const INIT: usize = 1;
```

Comment:
I'm afraid I'm not not familiar enough with all this code yet so I'll just make a nitpick suggestion: 1 << 0 ? :)

Reply:
I actually had that at first but Clippy suggested to change that to 1 (as its a nop) When Clippy tells me to do something, I usually do it =) 

BTW, we run Clippy as part of the CI so its suggestions, actually become failures. Thanks!